array/code/lib/lib.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  GetContactNumber: GetContactNumber,
+}
+
+var g_table = [
```

Comment:
can we rename this to contact_table or something more meaningful

Reply:
Done.